Moo/play/lib.py

```diff
@@ -108,13 +108,13 @@ def alpha(albums, mtag='artist'):
     return dict(letters)
 
 
-def album(key, index):
-    '''
-    returns album from index matching key
-    '''
-    for path in index:
-        if str(path).endswith(key):
-            return str(path)
+# def album(key, index):
+#     ''' Removing/Erasing
+#     returns album from index matching key
+#     '''
+#     for path in index:
+#         if str(path).endswith(key):
+#             return str(path)
```

Comment:
You commented out lib.album() which is what we want. That function is not being used, and is too close to lib.albums(). So, go ahead and delete that function. Don't worry, none of this code will be merged until we're ready.